.github/workflows/cypress.yml

```diff
@@ -0,0 +1,98 @@
+name: E2E Tests
+on: [push]
+jobs:
+  build:
+    name: E2E Tests
+    runs-on: self-hosted
+    defaults:
+      run:
+        working-directory: vets-website
+    env:
+      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem
+    steps:
+      - name: Checkout vets-website
+        uses: actions/checkout@v2
+        with:
+          path: vets-website
+
+      - name: Checkout vagov-content
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/vagov-content
+          path: vagov-content
+
+      - name: Checkout content-build
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/content-build
+          path: content-build
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ steps.get-node-version.outputs.NODE_VERSION }}
+
+      - name: Cache dependencies
+        id: cache-dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/yarn
+            node_modules
+          key: ${{ steps.get-node-version.outputs.NODE_VERSION }}-${{ hashFiles('yarn.lock') }}
+
+      - name: Install vets-website dependencies
+        run: yarn install --frozen-lockfile --prefer-offline
+        env:
+          YARN_CACHE_FOLDER: ~/.cache/yarn
+
+      - name: Install content-build dependencies
+        run: yarn install --frozen-lockfile --prefer-offline
+        working-directory: content-build
+        env:
+          YARN_CACHE_FOLDER: ~/.cache/yarn
+
+      - name: Build vets-website
+        run: yarn build:webpack --env.scaffold
+        timeout-minutes: 15
+        env:
+          NODE_ENV: production
+
+      - name: Fetch Drupal Cache
+        run: yarn fetch-drupal-cache
+        working-directory: content-build
+
+      - name: Build content-build
+        run: yarn build
+        working-directory: content-build
```

Comment:
I'm not sure. I can test it out. Would the command be `yarn build --env.scaffold`? I'm not seeing anything in the content-build README on using scaffold.

Reply:
It's the default now, so just a `yarn build` would suffice. It should work, since Jenkins is just doing the Webpack build right now. If it does, you can leave out both `vagov-content` and `content-build` from the build.

Reply:
Ah, I see what you mean. Testing it out right now.